src/main/java/com/linecorp/armeria/server/metrics/Status.java

```diff
@@ -0,0 +1,25 @@
+package com.linecorp.armeria.server.metrics;
+
+import com.linecorp.armeria.common.SessionProtocol;
+
+/**
+ * Server response status for {@link ServiceMetricConsumer}
+ */
+final class Status {
+    private final SessionProtocol sessionProtocol;
+    private final int code;
+
+    private Status(SessionProtocol sessionProtocol, int code) {
+        this.sessionProtocol = sessionProtocol;
+        this.code = code;
+    }
+
+    public static Status of(SessionProtocol protocol, int code) {
+        return new Status(protocol, code);
+    }
+
+    @Override
+    public String toString() {
+        return sessionProtocol.uriText() + '.' + code;
```

Comment:
Could we add a dedicated method such as `key()` which returns the string that can be used as a metrics registry key, so that we can change `toString()` implementation later when we ever change our mind to add more debug info there?